features/kinderen.feature

```diff
@@ -136,9 +147,9 @@ Functionaliteit: Kinderen van een persoon raadplegen
       | voornamen | Karel  |
 
   @proxy
-  Rule: Wanneer de geslachtsnaam van het kind onbekend is, wordt het geleverd met type "KindOnbekend" en indicatieOnbekend met waarde true
+  Rule: Wanneer de geslachtsnaam van het kind onbekend is, wordt er een 'kind' zonder gegevens geleverd.
```

Comment:
ik ben het niet eens met deze rule. wat als er geen geslachtsnaam is, maar wel voornaam, geboortedatum  en geslacht, waarom worden die niet geleverd?

Ik dacht dat we hebben afgesproken dat al deze speciale regels vervallen.

Reply:
Aangepast .